src/Umbraco.Web/Models/ContentEditing/MediaItemDisplay.cs

```diff
@@ -10,6 +10,10 @@ namespace Umbraco.Web.Models.ContentEditing
     [DataContract(Name = "content", Namespace = "")]
     public class MediaItemDisplay : ListViewAwareContentItemDisplayBase<ContentPropertyDisplay, IMedia>
     {
-        
+        [DataMember(Name = "documentType")]
+        public ContentPropertyDisplay DocumentType { get; set; }

```

Comment:
Same thing as above, `DocumentType` should be `MediaType` or we make them both `ContentType`